omnigibson/prims/rigid_prim.py

```diff
@@ -195,10 +196,10 @@ def update_meshes(self):
                     mesh.set_rest_offset(m.DEFAULT_REST_OFFSET)
                     self._collision_meshes[mesh_name] = mesh
 
-                    is_volume, volume, com = get_mesh_volume_and_com(mesh_prim)
-                    vols.append(volume)
-                    # We need to translate the center of mass from the mesh's local frame to the link's local frame
+                    volume, com = get_mesh_volume_and_com(mesh_prim)
```

Comment:
I think we should do this accumulation in the world frame too and then at the very end for the CoM we should apply the inverse scaled world transform for this object. Otherwise the volumes are not affected by the scales of any parents etc.

Reply:
Actually what you suggested won't work. Here we need the meshes CoM and volume in the *link* frame, not in the *world* frame. These computations are only for computing and setting the centerOfMass in the link frame, so I think my math is correct. But please sanity check me once again on this :D

Reply:
OK I see now that we don't return the volume and just use it for CoM computation. That should be fine. Thanks!